### DISCUSSION TITLE
How to configure online contact,like Daovoice, I install Next theme via npm, could u plz help me? wish ur reply soon, thx

### DISCUSSION THREAD

**Discussion by LukeLiou** [hexojs/hexo#5071] [Question / Answer] [answered]
## Check List

Please check followings before submitting a new issue.

- [ ] I have already read [Docs page](https://hexo.io/docs/) & [Troubleshooting page](https://hexo.io/docs/troubleshooting)
- [ ] I have already searched existing issues and they are not help  to me
- [ ] I examined error or warning messages and it's difficult to solve
- [ ] Using [the latest](https://github.com/hexojs/hexo/releases) version of Hexo (run `hexo version` to check)
- [ ] Node.js is higher than [minimum required version](https://hexo.io/docs/#Minimum-required-Node-js-version)

## How to reproduce?

* Step1
* Step2
* etc...

## Environment & Settings

**Node.js & npm version(`node -v && npm -v`)**

```
```

**Your site `_config.yml`** (Optional)

```yaml
```

**Hexo and Plugin version(`npm ls --depth 0`)**

```
```

**Your package.json `package.json`**

```
```

**Comment by yoshinorin** [comment]
@LukeLiou 
We don't know what is a `Daovoice`. But, perhaps you have to implement by yourself that like belows.

> https://github.com/jerryc127/hexo-theme-butterfly/blob/c48ce9af8c365372bcb55a61280005d0c24fb2fd/layout/includes/third-party/chat/daovoice.pug

BTW, we are not a help center and we can't answer 24/365. Also we don't know any unofficial plugins, themes and how to customize them.

**Comment by stevenjoezhang** [comment, marked as answer]
Please contact the developer of the theme you're using for help
